pkg/client/client.go

```diff
@@ -30,10 +30,13 @@ import (
 
 // Config returns a *rest.Config, using either the kubeconfig (if specified) or an in-cluster
 // configuration.
-func Config(kubeconfig, baseName string) (*rest.Config, error) {
-	loader := clientcmd.NewDefaultClientConfigLoadingRules()
-	loader.ExplicitPath = kubeconfig
-	clientConfig, err := clientcmd.BuildConfigFromKubeconfigGetter("", loader.Load)
+func Config(kubeconfig, kubecontext, baseName string) (*rest.Config, error) {
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+	loadingRules.ExplicitPath = kubeconfig
+	configOverrides := &clientcmd.ConfigOverrides{}
```

Comment:
Since you're only setting 1 field in here, how about this?
```go
configOverrides := &clientcmd.ConfigOverrides{CurrentContext: kubecontext}
```

Reply:
👍